var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -23,19 +23,39 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+from glob import glob
 
 
 class Libxsmm(Package):
-    '''LIBXSMM is a library for small dense and small sparse matrix-matrix
-    multiplications targeting Intel Architecture (x86).'''
+    '''Library targeting Intel Architecture
+    for small, dense or sparse matrix multiplications,
+    and small convolutions.'''
 
     homepage = 'https://github.com/hfp/libxsmm'
-    url      = 'https://github.com/hfp/libxsmm/archive/1.4.3.tar.gz'
+    url      = 'https://github.com/hfp/libxsmm/archive/1.7.1.tar.gz'
 
-    version('1.4.3', '9839bf0fb8be7badf1e97ce4c817149b')
-    version('1.4.2', 'ea025761437f3b5c936821b9ca21ec31')
-    version('1.4.1', '71648500ea4510529845d329091917df')
-    version('1.4',   'b42f91bf5285e7ad0463446e55ebdc2b')
+    version('develop', git='https://github.com/hfp/libxsmm.git')
+
+    version('1.7.1', 'a938335b1c2c90616dc72c2c1a5824ab')
+#   version('1.7',   'cb3aff6d123ba70bd3d4daf575767d14')
```

Comment:
I don't think there's a good reason to disable older versions. Spack won't choose them by default, so they don't inconvenience anybody. On the other hand, there are valid reasons for installing an older version, e.g. for reproducibility.